strawberry/contrib/starlette/app/graphql_app.py

```diff
@@ -69,16 +72,38 @@ def __call__(self, scope: Scope) -> ASGIInstance:
 
         background = BackgroundTasks()
         context = {"request": request, "background": background}
+        additional_arguments = {}
+        extensions = {}
+
+        if self.tracing:
```

Comment:
what if we allow users to pass the list of middlewares when instantiating the GraphQL app?

Reply:
ok, this might not be feasible since we need the tracing.start and .end

Reply:
I'm wondering if we should expose a way for users to be able to this themselves, I think for now let's keep this hard-coded, and we can improve it later ;)

Reply:
Ok, sounds good!